test/token/ERC20/SafeERC20.test.js

```diff
@@ -15,7 +15,7 @@ describe('SafeERC20', function () {
       this.wrapper = await SafeERC20Wrapper.new(hasNoCode);
     });
 
-    shouldRevertOnAllCalls('SafeERC20: call to non-contract');
+    shouldRevertOnAllCalls('Address: call to non-contract');
```

Comment:
@frangio are we ok with changing revert reasons in non-major releases?

(I think we are)

Reply:
Hm not possible to do that kind of thing dynamically. I don't think revert reasons should be stable, it's too limiting for us. I'm not 100% sure, but I think it should be ok to change it.

Reply:
If we _really_ wanted to keep the old message, we could have an extra `Address.isContract()` check on `SafeERC20` and have that fail first, but I agree with @frangio here and we can leave as-is.